File: typescript-angular/API/Client/CloudSearch.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface CloudSearch {

        "query"?: FilterBody;

        "sort"?: Array<string>;

        "collectionName"?: string;

        "key"?: string;

        "skip"?: number;

        "limit"?: number;
    }

}

File: akka-scala/src/main/scala/io/swagger/client/model/CloudSearch.scala
package io.swagger.client.model

import io.swagger.client.core.ApiModel
import org.joda.time.DateTime


case class CloudSearch (
  query: Option[FilterBody],
  sort: Option[Seq[String]],
  collectionName: Option[String],
  key: Option[String],
  skip: Option[Int],
  limit: Option[Int])
   extends ApiModel


